lib/miq_automation_engine/engine/miq_ae_service.rb

```diff
@@ -52,6 +59,14 @@ def initialize(ws)
       self.class.add(self)
     end
 
+    def enable_rbac
+      Thread.current.thread_variable_set(:miq_rbac, true)
+    end
```

Comment:
@kbrock 
This is a temporary hack for users to test with rbac enabled. At some point we are not going to give the user to enable/disable rbac. It would be enabled by default.